azerothcore/azerothcore-wotlk
# fix(Scripts/Temple of AhnQiraj): Moved Obsidian Nullifier script from…

… DB to C++.

Fixes #12876



## Issues Addressed:

- Closes #12876

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go c id 15312` with 2 or more mana users
see if it does not regenerate mana passively
see if it casts Cleave
see if it casts Drain Mana along with the visual spell every 6s
switch one mana user for one non-mana user, see if Drain Mana is not cast on them


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
